documentation/modules/proc-creating-kafka-cluster.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// assembly-evaluation.adoc
+
+[id='proc-kafka-cluster-{context}']
+= Creating a cluster
+
+Create a Kafka cluster, then a topic within the cluster.
+
+When you create a cluster, the Cluster Operator you deployed listens for new Kafka resources.
+
+.Prerequisites
+
+* For the Kafka cluster, a Cluster Operator is deployed
+* For the topic, a running Kafka cluster
+
+.Procedure
+
+. Log in to the Kubernetes cluster as a non-privileged user.
+
+. Create a new `my-cluster` Kafka cluster with 3 Zookeeper and 3 broker nodes.
++
+* Use `ephemeral` storage
+* Expose the Kafka cluster outside of the Kubernetes cluster using an external listener configured to use a `loadbalancer`.
++
+--
+[source,yaml,subs=attributes+]
+----
+cat << EOF | kubectl create -n my-kafka-project -f -
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+    listeners:
+      plain: {}
+      tls: {}
+      external:
+        type: loadbalancer
+    storage:
+      type: ephemeral
+  zookeeper:
+    replicas: 3
+    storage:
+      type: ephemeral
+  entityOperator:
+    topicOperator: {}
+EOF
+----
+--
+
+. Verify that the cluster was successfully deployed:
++
+[source,shell,subs="attributes+"]
+----
+kubectl get deployments
```

Comment:
I think we should use the status of the Kafka resource here. The `kubectl get deployments` would not return much useful.

Reply:
Do we want:

`kubectl rollout status -w my-cluster`
`kubectl describe deployment my-cluster`

Reply:
There is no single deployment which you can list. You can probably do somethinf like this what @tombentley used in the quickstarts:

```
kubectl wait kafka/my-cluster --for=condition=Ready --timeout=300s -n kafka 
```